### PR TITLE
Support prerelease branches in prepareRelease script

### PR DESCRIPTION
**Pull request by t1m0thyj**
After a Zowe Explorer release on the next branch, we want to push a commit that prepares for the next release (like we already do for other release branches).

Also moved the "run-script" package up into the `script` directory to make it easier to find 😋

### CODE REVIEW

**Review by traeok — APPROVED** [review]
LGTM, thanks @t1m0thyj

File: esbuild.mjs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import * as path from "path";
+import { fileURLToPath } from "url";
 import * as esbuild from "esbuild";
 
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
```
Comment (zFernand0) [inline esbuild.mjs:5]: This is clever!

General question, would you consider using `import.meta.url` a better way to get which module was loaded compared to throwing an error end search the stacktrace?

**Review by zFernand0 — APPROVED** [review]
lots of moving parts, but the changes make sense to me! 🥳 
LGTM! 😋

File: esbuild.mjs (hunk 1 above)
Comment (t1m0thyj) [reply esbuild.mjs:5]: It's an alternative to using `__dirname` which doesn't work in .mjs files. I don't know if it's much better though 😋

**Review by traeok — APPROVED** [review]

**Pull request merged**